code/modules/codex/categories/category_recipes.dm

```diff
@@ -0,0 +1,58 @@
+/datum/codex_category/recipes
+	name = "Recipes"
+	desc = "Recipes for a variety of different kinds of foods and condiments."
+	guide_name = "Cooking"
+
+/datum/codex_category/recipes/Populate()
+
+	var/list/entries_to_register = list()
+
+	guide_html = {"
+		<h1>Chef recipes</h1>
+		Here is a guide on food recipes and also how to not poison your customers accidentally.
+
+		<h3>Basics:</h3>
+		<ul>
+		<li>Mix an egg and some flour along with some water to make dough.</li>
```

Comment:
Make sure this is also accurate to whatever version of TG cooking we're on right now.